pkg/realmd/operation.html

```diff
@@ -8,6 +8,15 @@ <h4 class="modal-title realms-op-title"></h4>
         <div class="realms-op-fields">
           <p class="realms-op-leave-only-row"></p>
           <table class="form-table-ct realms-op-form">
+            <tr class="realms-op-error" hidden>
+              <td colspan="3">
+                <div class="alert alert-danger">
+                  <span class="pficon pficon-error-circle-o"></span>
+                  <strong class="realms-op-message"></strong>
+                </div>
+                <pre class="realms-op-diagnostics" hidden></pre>
+              </td>
+            </tr>
```

Comment:
It now looks fine overall, although I think we should probably move the error message back down to the bottom before this is merged, similar to what's in the Cockpit HIG:

https://github.com/cockpit-project/cockpit/wiki/HIG:-Dialogs#example

Here's a direct quote:

> Show error messages directly below the input fields

(And there are wireframes on that page too.)